service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -307,19 +353,19 @@ void testUpdateWorkspace() {
             USER_REQUEST, workspaceUuid, null, null, otherDescription, null);
 
     // Since name is null, leave it alone. Description should be updated.
-    assertEquals(userFacingId, secondUpdatedWorkspace.getUserFacingId().orElse(null));
-    assertEquals(name, secondUpdatedWorkspace.getDisplayName().orElse(null));
-    assertEquals(otherDescription, secondUpdatedWorkspace.getDescription().orElse(null));
+    assertEquals(name, secondUpdatedWorkspace.getDisplayName().get());
+    assertEquals(otherDescription, secondUpdatedWorkspace.getDescription().get());
     assertEquals(propertyMap2, updatedWorkspace.getProperties());
 
     // Sending through empty strings and an empty map clears the values.
     Map<String, String> propertyMap3 = new HashMap<>();
     Workspace thirdUpdatedWorkspace =
         workspaceService.updateWorkspace(
             USER_REQUEST, workspaceUuid, userFacingId, "", "", propertyMap3);
-    assertEquals("", thirdUpdatedWorkspace.getDisplayName().orElse(null));
-    assertEquals("", thirdUpdatedWorkspace.getDescription().orElse(null));
+    assertEquals("", thirdUpdatedWorkspace.getDisplayName().get());
```

Comment:
nit: never call `get()` without checking/asserting `isPresent()`. Even in test code, it makes me look up to see if you've checked it.

Reply:
Yes it will fail, but not with your assertion failure, so if you have a description in your assertion, you won't see it. One of these libraries has `assertContains(T value, Optional<T> o)` that lets you do it without the get.

Reply:
Ah, it makes sense to assert `isPresent()` to get a friendlier error message. Done.